LanguageFeatures/Extension-types/static_analysis_extension_types_A02_t01.dart

```diff
@@ -22,19 +22,19 @@ extension type ET1(int id) {
 // [analyzer] unspecified
 // [cfe] unspecified
 
-  ET1.n2(int id) : this();
+  ET1.n2(int id) : this(id);
```

Comment:
This case doesn't look like it's a compile-time error. Constructor `ET1.n2` declares a formal parameter, `id`, and passes it as the argument to the constructor that initializes the field with the same name. By analogy, the following Dart code compiles and runs without errors:

```dart
class A {
  int foo;
  A(this.foo);
  A.named(int foo): this(foo);
}

main() {}
```

/cc @eernstg

Reply:
I also don't see any compile-time errors here.